stackslib/src/chainstate/stacks/mod.rs

```diff
@@ -658,7 +658,13 @@ pub struct SchnorrThresholdSignature {
     //pub scalar: wsts::Scalar,
 }
 
-/// Reasons why a `TenureChange` transaction can be de
+impl SchnorrThresholdSignature {
```

Comment:
Oh I didn't see this when i was building off this branch. Perhaps I should be using this then instead of my SchnorrSignature in my own PR.

Reply:
I think this might be a @jbencin and @xoloki question. IIRC @xoloki told him to use `wsts::Point` and `wsts::Scalar` to encode the signature, but that was a while ago. It's possible that `wsts::common::Signature` is more appropriate now.

Reply:
You should do whatever makes the most sense in your PR :)  This PR only needs this struct to encode/decode; it doesn't care about validating the contents.

Reply:
`ThresholdSignature` is the same as `wsts::common::Signature`, I defined a struct in this repo so that I could implement `StacksMessageCodec` for it (can't do that with types defined outside this crate)